src/OMSimulatorLib/SystemTLM.cpp

```diff
@@ -133,6 +134,75 @@ oms_status_enu_t oms3::SystemTLM::terminate()
   return logError_NotImplemented;
 }
 
+oms_status_enu_t oms3::SystemTLM::connectToSockets(const oms3::ComRef cref, std::string server)
+{
+  logInfo("Starting TLM simulation thread for model "+std::string(cref));
+
+  SystemWC* system = reinterpret_cast<SystemWC*>(getSystem(cref));
+  if(system == nullptr)
+    return logError_SubSystemNotInSystem(getName(),cref);
+
+  ///< Todo: Where obtain time and communication interval?
+  double time = 0;    ///< Todo: Remove when implemented
+  double communicationInterval = 0.001;   ///< Todo: Remove when implemented
```

Comment:
@lochel Will time and communicationInterval be implemented soon, or should I merge this as it is and fix this later? This PR does not break any existing functionality and is needed for implementing other TLM functions.

Reply:
@robbr48 I will push my changes today, which contain `time` and `communicationInterval`.

Reply:
Perfect!